src/Kmd.Logic.FileSecurity.Client/FileSecurityClient.cs

```diff
@@ -131,6 +131,33 @@ public async Task<HttpResponseMessage> DeleteCertificate(Guid certificateId)
             }
         }
 
+        /// <summary>
+        /// Get sign configuration.
+        /// </summary>
+        /// <param name="signConfigurationId">SignConfigurationID.</param>
+        /// <returns>SignConfigurationPdfResponse.</returns>
+        public async Task<SignConfigurationPdfResponse> GetSignConfiguration(Guid signConfigurationId)
```

Comment:
Methos name should be `GetPdfSignConfiguration`.

Reply:
Method name renamed to GetPdfSignConfiguration